cosmic-de/cosmic-wallpapers/cosmic-wallpapers-1.0.0_alpha6.ebuild

```diff
@@ -11,6 +11,19 @@ HOMEPAGE="https://github.com/pop-os/cosmic-wallpapers"
 inherit git-r3
 EGIT_REPO_URI="${HOMEPAGE}"
 EGIT_COMMIT="epoch-1.0.0-alpha.6"
+# Due to the way things are setup at the moment, we reuse the same `DISTDIR`
```

Comment:
Please point the comment to the COSMIC_GIT_UNPACK eclass variable in the eclass :)

Reply:
Or better yet... I'd just say something along the lines of "see use of EGIT_CLONE_TYPE and EGIT_LFS_CLONE_TYPE in the cosmic-de eclass"